builders/integrations/oracles/api3.md

```diff
@@ -136,96 +142,47 @@ The data feeds are continuously updated by [first-party oracles](https://docs.ap
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-6.webp)
-
-Use the gas estimator to select how much gas is needed by the dAPI. Click on **Send DEV** to send the entered amount to the sponsor wallet of the respective dAPI.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-7.webp)
-
-Once the transaction is broadcasted & confirmed on the blockchain a transaction confirmation screen will appear.
+| Deviation | Heartbeat |
+| --------- | --------- |
+| 0.25%     | 24 hours  |
+| 0.5%      | 24 hours  |
+| 1%        | 24 hours  |
+| 5%        | 24 hours  |
 
-#### Deploy a Proxy Contract to Access The dAPI {: #deploy-proxy }
+![API3 Market 1](/images/builders/integrations/oracles/api3/dapi-1.png)
 
-Smart contracts can interact and read values from contracts that are already deployed on the blockchain. By deploying a proxy contract via the API3 Market, a dApp can interact and read values from a dAPI like ETH/USD.
+#### Activating your dAPI {: #select-a-dapi }
 
 !!! note
-    If a proxy is already deployed for a self-funded dAPI, the dApp can read the dAPI without having to deploy a proxy contract. They do this by using the address of the already deployed proxy contract which will be visible on the API3 Market.
-
-If you are deploying a proxy contract during the funding process, clicking on the **Get proxy** button will initiate a transaction to your wallet that will deploy a proxy contract.
-
-![API3 Remix deploy](/images/builders/integrations/oracles/api3/api3-8.webp)
-
-Once the transaction is broadcasted & confirmed on the blockchain, the proxy contract address will be shown on the UI.
+    If a dAPI is already activated, make sure to check the expiration date and update parameters. You can update the parameters and extend the subscription by purchasing a new configuration.
 
-### Access Managed dAPIs {: #access-managed-dapis }
+After selecting the dAPI and the configuration, you will be presented with an option to purchase the dAPI and activate it. Make sure to check the time and amount of the subscription. If everything looks good, click on **Purchase**.
 
-If you are trying to access managed dAPIs, once you have selected your dAPI, you will then be presented with the option to choose from either **Managed** or **Self-funded**. Select **Managed dAPIs**.
+![API3 Market 2](/images/builders/integrations/oracles/api3/dapi-2.png)
 
-Managed dAPIs give you the option to configure the dAPI's [devation threshold](https://docs.api3.org/reference/dapis/understand/deviations.html){target=\_blank} and [heartbeat](https://docs.api3.org/reference/dapis/understand/deviations.html#heartbeat){target=\_blank}. For managed dAPIs, you will have the following options to choose from:
+You can then connect your wallet and confirm the transaction. Once it's confirmed, you will be able to see the updated configuration for the dAPI.
```

Comment:
Updated configuration? So does a free configuration exist? Is that why I am getting confused haha